packages/@dcl/inspector/src/lib/babylon/decentraland/GizmoManager.ts

```diff
@@ -385,6 +385,14 @@ export function createGizmoManager(context: SceneContext) {
           currentTransformer.setup()
           currentTransformer.setEntities(selectedEntities)
 
+          // Pass GizmoManager reference to FreeGizmo for centroid calculation
+          if ('setGizmoManager' in currentTransformer) {
```

Comment:
wdyt about adding this method `setGizmoManager` to the `IGizmoTransformer` as optional or passing it in the constructor of the `FreeGizmo` class?